src/js/components/DataTable/stories/Fill.js

```diff
@@ -42,9 +43,7 @@ export const Fill = () => (
         step={10}
         fill
         pin
-        background={{
-          pinned: { color: 'background-contrast' },
-        }}
+        background={{ body: 'light-1' }}
```

Comment:
interesting test case but we may not want the example to promote setting the background this way